pilot/lib/pilot/manager.rb

```diff
@@ -17,12 +17,22 @@ def start(options, should_login: true)
     end
 
     def login
-      config[:username] ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
+      if api_token
+        UI.message("Creating authorization token for App Store Connect API")
+        Spaceship::ConnectAPI.token = api_token
+      else
+        config[:username] ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
 
-      UI.message("Login to App Store Connect (#{config[:username]})")
-      Spaceship::Tunes.login(config[:username])
-      Spaceship::Tunes.select_team(team_id: config[:team_id], team_name: config[:team_name])
-      UI.message("Login successful")
+        UI.message("Login to App Store Connect (#{config[:username]})")
+        Spaceship::ConnectAPI.login(config[:username], team_id: config[:team_id], team_name: config[:team_name])
```

Comment:
The change here from iTunes login to connect api login is causing a regression where passing team_id doesn't work anymore since App Store connect login uses both Dev portal and iTunes connect login with the same team Id which aren't the same. 

Pilot works fine in 2.156.1 but there is no team Id which will work with v2.157.0.